README.md

```diff
@@ -87,6 +87,45 @@ Full documentation can be found on the [Lisk documentation site][].
 | [@liskhq/lisk-passphrase](packages/lisk-passphrase)     |  [![](https://img.shields.io/badge/npm-v1.1.0-green.svg)](https://www.npmjs.com/package/@liskhq/lisk-passphrase)  | Mnemonic passphrase helpers for use with Lisk-related software     |
 | [@liskhq/lisk-transactions](packages/lisk-transactions) | [![](https://img.shields.io/badge/npm-v1.1.0-green.svg)](https://www.npmjs.com/package/@liskhq/lisk-transactions) | Everything related to transactions according to the Lisk protocol  |
 
+## Tests
```

Comment:
maybe we should add `Build` too?

Reply:
`npm run build` is already included in the installation section